kotlin-source/build.gradle

```diff
@@ -78,7 +87,11 @@ task deployNodes(type: net.corda.plugins.Cordform, dependsOn: ['jar']) {
     node {
         name "O=Notary,L=London,C=GB"
         notary = [validating : false]
-        p2pPort 10006
+        p2pPort 10005
+        rpcSettings {
```

Comment:
Would you explain why it was necessary to specify RPC for Notary?
(Notmally Notary node doesn't accept RPC connections)

Reply:
Because otherwise it gave me the following error:
```
[ERROR] 17:59:10+0200 [main] subcommands.ValidateConfigurationCli.logConfigurationErrors$node - Error(s) while parsing node configuration:
	- for path: "<unknown>": No configuration setting found for key 'rpcSettings.address'
```
Probably this is a bug in Corda 4 similar to finance app config, where there are not good defaults or this is some intended change. I'm not sure.

Reply:
@blsemo and @thschroeter - Do you know if is it a known thing that `rpcSettings.address` has to be present in V4 even for Notary nodes?

Reply:
@vkolomeyko I changed P2P port because all ports was in order and I don't want to break the order. One option was to change P2P port of the Notary or to change all ports of all other nodes. I decided that changing one value is less invasive.

I will wait for the change in the `java-source` until we have an answer if making `rpcSettings.address` required for all nodes is intentional. If it was not I need to revert the changes in `kotlin-source/build.gradle`

Reply:
Sorry for the delay. Changes are pushed.